graphql-parser/src/lib.rs

```diff
@@ -20,33 +20,24 @@
 //! -------------------------------
 //!
 //! ```rust
-//! # extern crate graphql_parser;
 //! use graphql_parser::{parse_query, ParseError};
 //!
-//! # fn parse() -> Result<(), ParseError> {
-//! let ast = parse_query("query MyQuery { field1, field2 }")?;
+//! let ast = parse_query("query MyQuery { field1, field2 }").unwrap();
 //! // Format canonical representation
 //! assert_eq!(format!("{}", ast), "\
 //! query MyQuery {
 //!   field1
 //!   field2
 //! }
 //! ");
-//! # Ok(())
```

Comment:
yeah, the `#` hides the line from the documentation, but _compiles_ the code with it. Because this was inside a function (whose signature was also hidden with a `#`) that expected a `Result` return type, this was added to make it compile.